tests/test_get_hosts.py

```diff
@@ -0,0 +1,68 @@
+import unittest
```

Comment:
Let's not use `unittest`. Pytest is prefered, see https://github.com/ManageIQ/aiops-incoming-listener/blob/master/tests/test_app.py

Reply:
@tumido is this a blocker for this PR? `pytest` looks great, I'll follow up with another PR for that.
Would like to get his in as it's blocking an end to end trip of the outlier detection pipeline